src/aliceVision/fuseCut/DelaunayGraphCut.cpp

```diff
@@ -1078,6 +1078,13 @@ void DelaunayGraphCut::fuseFromDepthMaps(const StaticVector<int>& cams, const Po
             pixSizePrepare[vIndex] = -1;
             continue;
         }
+        // Filter points based on their number of observations
+        if(visCams.size() < params.minVis)
+        {
+            ALICEVISION_LOG_DEBUG("minVis value:" << params.minVis);
```

Comment:
Remove this log, add a counter, and log at the end the number of points filtered based on the number of observations.